linode/domains/framework_schema.go

```diff
@@ -0,0 +1,45 @@
+package domains
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/linode/terraform-provider-linode/v2/linode/domain"
+	"github.com/linode/terraform-provider-linode/v2/linode/helper"
+	"github.com/linode/terraform-provider-linode/v2/linode/helper/frameworkfilter"
+)
+
+var filterConfig = frameworkfilter.Config{
+	"group": {APIFilterable: true, TypeFunc: helper.FilterTypeString},
+	"tags":  {APIFilterable: true, TypeFunc: helper.FilterTypeString},
+
+	"domain":      {APIFilterable: false, TypeFunc: helper.FilterTypeString},
```

Comment:
Can it be properly filtered by API? I saw it's marked as API filterable on [the API doc](https://www.linode.com/docs/api/domains/#domains-list__responses).

Reply:
Yes it's filterable. Was a mistake mark it as false.